js/index.js

```diff
@@ -37,6 +37,140 @@ const siteContent = {
   },
 };
 
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// borders
+let borders = document.querySelector(".main-content");
+borders.style.borderColor = "dodgerblue";
+borders.style.borderStyle = "dashed";
+
+// Nav items
+let navitems = document.querySelectorAll('nav > a');
+navitems[0].textContent = siteContent["nav"]["nav-item-1"];
+navitems[1].textContent = siteContent["nav"]["nav-item-2"];
+navitems[2].textContent = siteContent["nav"]["nav-item-3"];
+navitems[3].textContent = siteContent["nav"]["nav-item-4"];
+navitems[4].textContent = siteContent["nav"]["nav-item-5"];
+navitems[5].textContent = siteContent["nav"]["nav-item-6"];
+
+
+// appending and prepending links 
+
+let nav = document.querySelector("nav");
+
+let appendLink = document.createElement('a');
+appendLink.textContent = "Appended Item";
+nav.appendChild(appendLink);
+
+let prependLink = document.createElement('a');
+prependLink.textContent = "Prepended Item";
+nav.prepend(prependLink);
+
+let navitems2 = document.querySelectorAll("a");
+
+navitems2.forEach(function (a) {
+  a.style.color = "dodgerblue";
+  a.style.fontWeight = "bold";
+})
+
+// CTA section
+
+let ctaImage = document.getElementById("cta-img");
+ctaImage.setAttribute('src', siteContent["cta"]["img-src"]);
+ctaImage.style.border = "3px dodgerblue solid"
+ctaImage.style.borderRadius = "50%"
+ctaImage.style.opacity = "1%";
+
+let ctaH1 = document.querySelector(".cta h1");
+ctaH1.innerHTML = "DOM<br> IS<br> AWESOME";
+
+
+
+let ctaBtn = document.querySelector(".cta button");
+ctaBtn.textContent = siteContent["cta"]["button"];
+ctaBtn.style.boxShadow = "5px 5px 5px dodgerblue";
+ctaBtn.style.borderRadius = "1rem"
+ctaBtn.style.border = "1px dodgerblue solid"
+ctaBtn.style.color = "dodgerblue"
+
+
+// ----------- an extra button trying to enable/disable an animation. It didn't work!
```

Comment:
You were really close to getting this to work! I think after the event listener lecture today you could definitely play around with this to get it going!